### ISSUE TITLE
showing just annotated tokens as previously annotated

### ISSUE THREAD

**Issue by odanoburu** [own-pt/sensetion.el#22]
the interface shows tokens that were annotated in the current iteration in a different color from those who had already been annotated.

however there's a bug where a subsequent non-related annotation (such as the unglobbing of a token) will reset the color to that of a previously annotated token.

**Comment by arademaker** [comment]
I need more details about the second paragraph. But that will be more clear once I can test the interface.

**Comment by odanoburu** [comment]
yes, I'll show you. but here's concrete example: after a query, we show previously annotated tokens in green. when the user annotates a token, it is shown in blue. but if the user annotates another token in the same sentence, the first token becomes green.

**Comment by arademaker** [comment]
Hum, so we don't have the concept of transaction was we had discussed . Fine for the first version.
